frontend/index.html

```diff
@@ -24,13 +39,21 @@
 
       // Very early load of theme definition to avoid flickering
       document.addEventListener('DOMContentLoaded', () => window.refreshTheme())
+
+      // Template variable from Golang process
```

Comment:
I like this more than the `/vars.js` endpoint. It breaks something for me, though: I prefer to allow browsers/cloudflare to cache the complete webapp (as especially the js is quite big). Having the language code in the `index.html` means I can't add a permissive Cache-Control header for index.html anymore.

I think it should be possible to get rid of the language code completely and use something like `locale: navigator.language.split('-')[0]` instead. ([source](https://github.com/kazupon/vue-i18n/issues/220), I don't know Vue).

There's also support for changing the language using a cookie, but it looks like there's no way to actually set it. I [used the cookie during testing](https://github.com/sorcix/ots/commit/42a1ee3c857fc981d631c476193ce1b5ec6b14a5) and it does work. Is the cookie support in there to be compatible with some external system? Unless it's a http-only cookie, I guess it would be possible to read it using Javascript too.

Reply:
The cookie indeed is for testing purposes… One could implement a language switch but currently it's not planned to do so. I'm using the cookie to test translations without having to modify my browser language…

I've moved the language detection to the frontend. So now only the version variable is passed into the template together with the SRI hashes and should be removed from the cache when updating the application. (Which has to be done either way to have matching SRI hashes…)